.github/CONTRIBUTING.md

```diff
@@ -111,20 +111,20 @@ These instructions assume a parent directory `bar` and we will be adding two chi
 4. Continue running commands in the **botpress terminal window**
     ```shell
     # Install the dependencies
-    bar/botpress $ yarn install
+    botpress/packages/core/botpress $ yarn install
```

Comment:
I think instead one should run `./node_modules/.bin/lerna bootstrap` in the repo root to also use the most recent (master) versions of every dependency, properly interlinked.